src/RahulH/CoreJavaAssignment3/ClassesAndObjects/Addition.java

```diff
@@ -0,0 +1,26 @@
+package RahulH.CoreJavaAssignment3.ClassesAndObjects;
+
+public class Addition {
+	
+	int isum;
+	double dsum;
+	String ssum;
+	
+	void AddTwoIntegers(int i1, int i2)
+	{
+		isum = i1 + i2;
+		System.out.println("Total integer sum is: " +isum);
+	}
+
+	void AddTwoDecimal(double d1, double d2)
+	{
+		dsum = d1 + d2;
+		System.out.println("Total double sum is: " +dsum);
+	}
+	
+	void AddTwoString(String s1, String s2)
```

Comment:
wrong it will give wrong output
like if i pass "12" & "10" then it will print 1210
think